pkg/sbom/cyclonedx/unmarshal.go

```diff
@@ -434,3 +431,30 @@ func packageName(typ, pkgNameFromPurl string, component cdx.Component) string {
 	}
 	return component.Name
 }
+
+// getPackageLicenses checks all supported license fields and returns a list of licenses.
+// https://cyclonedx.org/docs/1.5/json/#components_items_licenses
+func getPackageLicenses(l *cdx.Licenses) []string {
```

Comment:
nit: A more detailed function name is preferred, as `get` does not contain much information in most cases.

```suggestion
func parsePackageLicenses(l *cdx.Licenses) []string {
```

Even for getters, `GetXXX` is not common in Go.
https://go.dev/doc/effective_go#Getters

Reply:
changed in https://github.com/aquasecurity/trivy/pull/5828/commits/a79bcd4ca45ca71dc03b38ef5b2a2e2a1aac3ef2